src/region_growing/label_connected_comp.py

```diff
@@ -103,6 +117,48 @@ def _fill_components(self, threshold=0.1):
 
         return label_mask, points_added
 
+    def _fill_car_like_components(self, road_polygons, ahn_tile):
+        """ Label car like clusters.  """  # TODO text
+        mask_indices = np.where(self.mask)[0]
+        label_mask = np.zeros(len(self.mask), dtype=bool)
+
+        cc_labels, counts = np.unique(self.point_components,
+                                      return_counts=True)
+
+        cc_labels_filtered = cc_labels[counts >= self.min_component_size]
+
+        surface = ahn_tile['ground_surface']
+        fast_z = FastGridInterpolator(ahn_tile['x'], ahn_tile['y'], surface)
+
+        for cc in cc_labels_filtered:
+            # select points that belong to the cluster
+            cc_mask = (self.point_components == cc)
+
+            target_z = fast_z(self.las[mask_indices[cc_mask]])
+            valid_values = target_z[np.isfinite(target_z)]
+
+            if valid_values.size != 0:
+                max_z_thresh = np.mean(valid_values) + self.max_above_ground
```

Comment:
Misschien makkelijker zo:
```
max_z_thresh = np.nanmean(target_z)
if not np.isnan(max_z_thresh):
    ...
```

Reply:
Nanmean gaf runtime warning en "warning: Mean of empty slice" dus heb deze code ervan gemaakt.

Reply:
Ahja, ik snap het.